src/entities.rs

```diff
@@ -191,6 +191,17 @@ pub(crate) struct Entities {
     len: u32,
 }
 
+impl Clone for Entities {
+    fn clone(&self) -> Self {
+        Self {
+            meta: self.meta.clone(),
+            pending: self.pending.clone(),
+            free_cursor: AtomicIsize::new(self.free_cursor.load(Ordering::Relaxed)),
```

Comment:
I wonder if this should be played safe and the concurrently modifiable state should not be cloned? For example, would pending updated applied to both worlds?

Reply:
I agree on the `free_cursor` and I do not fear undefined behaviour here, but rather surprising behaviour if there are pending updates which still need to be applied.

> Perhaps it might be worth making World::try_clone() error if the World needs to be flushed (probably via replacing Entities::clone() with Entities::try_clone(), on the assumption that cloning a world that needs to be flushed is probably a mistake on the part of the user? (I don't understand exactly when World::flush() (which is pub) would be expected to be called by the user, so I can't really judge whether there's a valid use case for cloning an unflushed world)

Yes, I think erring out if the world has pending changes (and documenting that requirement) would be a more ergonomic API.